java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -3538,4 +3538,37 @@ public void bitcount(BaseClient client) {
             assertTrue(executionException.getCause() instanceof RequestException);
         }
     }
+
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void setbit(BaseClient client) {
+        String key1 = UUID.randomUUID().toString();
+        String key2 = UUID.randomUUID().toString();
+
+        assertEquals(1, client.sadd(key2, new String[] {"value"}).get());
```

Comment:
```suggestion
        assertEquals(1, client.sadd(key1, new String[] {"value"}).get());
```

Reply:
why `sadd`?

Reply:
or move down...

Reply:
key2 needs to be a set to check for wrong type. I'll move down.